test/basic.test.js

```
@@ -21,7 +21,9 @@ describe('basic', function () {
     components.forEach((cm) => {
       var specfile = testfolder + '/' + cm.replace(/\.vue$/, '.spec.js')
       var exists = Fs.existsSync(specfile)
-      expect(exists ? cm : 'not-found:' + specfile).equal(cm)
+      expect(exists ? cm.replace('.vue', '') : 'not-found:' + specfile).equal(
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)